src/Configuration/Properties/Property.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelCom\Integration\Configuration\Properties;
+
+class Property
```

Comment:
I think we no longer talk about properties here, as in json-schema properties. Furthermore, it's good to make sure no one can instantiate this class straight away by making it abstract. 

A more solid argumentation about the rename of this class to _Field_ I wrote in a comment below.

```suggestion
abstract class Field
```